### DISCUSSION TITLE
Adding GitHub Actions

### DISCUSSION THREAD

**Discussion by lianaling** [LianaLing/art-gallery-website#4] [Ideas]
I'm thinking if we need to add an automated workflow. See [here](https://docs.github.com/en/actions/quickstart) for reference.

Is there anything we need to automate considering our website?

If you @marcustut find it useful, can you let me try setting it up? I'm curious about it.

**Comment by marcustut** [comment]
Yea adding github actions is definitely a good thing to do but the problem is that GitHub Actions uses linux under the hood and I don't seem to find a way to run the project in Linux, so unless Github Actions is able to run in a Windows environment else it's mostly pointless
